k8s-daemonset/k8s/linkerd-rbac-beta.yml

```diff
@@ -10,7 +10,7 @@ rules:
     resources: ["endpoints", "services", "pods"] # pod access is required for the *-legacy.yml examples in this folder
     verbs: ["get", "watch", "list"]
 ---
-# grant namerd permisisons to third party resources for dtab storage
+# grant namerd permissions to custom resource definitions for dtab storage
```

Comment:
since the contents of this file are identical to `linkerd-rbac-beta-legacy.yml`, i'd say just drop that other file, and update this comment to call out TPR for k8s <1.8, CRD for >=1.8